perf_dashboard/artifacts/views.py

```diff
@@ -16,6 +16,8 @@
 from helpers import download
 import os
 
+current_release = [os.getenv('CUR_RELEASE')]
```

Comment:
just curious, is current_release supposed to be a list?

Reply:
yes, it should be casted to a list in order for template to parse view's context.